install/k8s/cluster/bootstrap_centos.sh

```diff
@@ -38,14 +38,16 @@ EOF
 
 sysctl --system
 
-yum remove -y docker \
-                  docker-common \
-                  container-selinux \
-                  docker-selinux \
-                  docker-engine \
-                  docker-engine-selinux
+yum remove -y docker docker-common container-selinux docker-selinux docker-engine docker-engine-selinux
+yum install -y docker
```

Comment:
curious why docker was removed above just to be reinstalled, is there something in pre/post installation scripts?